Kernel/Memory/Region.cpp

```diff
@@ -505,7 +506,7 @@ PageFaultResponse Region::handle_zero_fault(size_t page_index_in_region, Physica
     RefPtr<PhysicalRAMPage> new_physical_page;
 
     if (page_in_slot_at_time_of_fault.is_lazy_committed_page()) {
-        new_physical_page = static_cast<AnonymousVMObject&>(*m_vmobject).allocate_committed_page({});
+        new_physical_page = anon_vmobject.allocate_committed_page({});
```

Comment:
This change and the change below look more like a general refactoring, maybe move them into a separate commit

Reply:
Ok, I moved those changes into a new commit